### PR TITLE
Wires fuses

### PR DESCRIPTION
**Pull request by RhushabhRR**
Content on Wires and Fuses added

### CODE REVIEW

File: docs/system/wires_fuses.md [hunk 1]
```diff
@@ -2,15 +2,84 @@
...
+| Material          | Specific Resistance $\rho$ ($\Omega$ mm2/m)  |
+|:-----------------:|:--------------------------------------------:|
+|   Silver          |                  0.0159                      |
+|   Copper          |                  0.0168                      |
+| Copper,annealed   |                  0.0172                      |
+|   Iron            |                  0.0971                      |
+
+### 4. Ambient temperature
+
+Higher the ambient temperature, less heat (generated by current flowing through wire) will be sufficient for the insulation to melt. Hence ambient temperature has to be kept low to achieve higher current carrying capacity of the wire. The deviation of current carrying capacity (from 30C) depending on ambient temperature is given in the below table, which is according to the VDE 0298 norm.
```
Comment (martinjaeger) [inline docs/system/wires_fuses.md]: `°C` is the correct unit. `C` is coulombs.

File: docs/system/wires_fuses.md [hunk 2]
```diff
@@ -2,15 +2,84 @@
...
+
+### 2. Length
+
+Higher the length of the conductor, greater is the effective resistance offered by the path and hence more power will be dissipated. This limits the maximum current carrying capacity of the conductor and reduces its efficiency as well.
+
+### 3. Specific resistance
+
+Specific resistance is the property of the material used for the wire. Different materials like copper, aluminium, silver and others exhibit different value of specific resistance. Materials like silver with low specific resistance are more conductive in nature, but are more expensive as well. Hence wire of suitable material should be used according to the requirement. The following table gives the value of specific resistance of few materials.
+
+| Material          | Specific Resistance $\rho$ ($\Omega$ mm2/m)  |
```
Comment (martinjaeger) [inline docs/system/wires_fuses.md]: The mm2/m should also be included into LaTeX syntax to be rendered more nicely.

File: docs/system/wires_fuses.md [hunk 3]
```diff
@@ -2,15 +2,84 @@
...
+|   Silver          |                  0.0159                      |
+|   Copper          |                  0.0168                      |
+| Copper,annealed   |                  0.0172                      |
+|   Iron            |                  0.0971                      |
+
+### 4. Ambient temperature
+
+Higher the ambient temperature, less heat (generated by current flowing through wire) will be sufficient for the insulation to melt. Hence ambient temperature has to be kept low to achieve higher current carrying capacity of the wire. The deviation of current carrying capacity (from 30C) depending on ambient temperature is given in the below table, which is according to the VDE 0298 norm.
+
+| Ambient Temperature (C)   | Deviating Factor   |
```
Comment (martinjaeger) [inline docs/system/wires_fuses.md]: Also degree C.

And is it a derating factor? Deviating factor sounds strange to me. Or may deviation factor?

File: docs/system/wires_fuses.md [hunk 4]
```diff
@@ -2,15 +2,84 @@
...
+</figure>
 
 ## Fuses
 
-::: warning TODO
-- General rule: fuse always at the energy source, explain reason...
-- Star vs. mesh structure and influence on fuses
-- Electronic fuses
-:::
+Fuse is an important component in electrical systems, used to prevent excessive flow of current which can lead to damage of electrical components or even accidents like fire. Proper care must be taken in deciding the placement of the fuse to prevent any damage. Generally it is always advisable to place the fuse as close as possible to the energy source. If the fuse is placed away from the energy source as shown in Fig.2a, the short-circuit which occurs at point X would go undetected, resulting in excessive current. Whereas in the case of Fig.2b, this problem can be avoided.
```
Comment (martinjaeger) [inline docs/system/wires_fuses.md]: Article missing: A fuse...

**Review by martinjaeger — CHANGES_REQUESTED** [review]
Only some tiny style changes requested, see below.

**Comment by RhushabhRR** [comment]
Tiny style changes fixed.

**Review by martinjaeger — APPROVED** [review]

**Pull request merged**